### PR TITLE
Add token examples of python

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <rxl@apache.org>

- Add token examples of python
- Add pulsarctl examples
- Add pulsar-admin examples
- Add pulsar-client examples
- Add pulsar-perf examples
- Add token examples of nodejs
- Update readme

### CODE REVIEW

File: cloud/README.md [hunk 1]
```diff
@@ -2,15 +2,84 @@
...
+For the OAuth2, it needs to obtain the specified options from the Pulsar cluster and OAuth2 services. [How to get OAuth2 options](#How to get OAuth2 options) explains how you get these options.
 
 Currently, we support the following three languages to connect through OAuth2:
 
 - Java
 - Go
 - CPP
 
-For clients in other languages, you can connect through token, reference to [here](https://pulsar.apache.org/docs/en/security-tls-transport/#client-configuration).
+For the Token, we support the following languages, and [How to get Token options](#How to get Token options) explains how you get these options.
```
[the remark this answers is not in the document]
Comment (wolfstudy) [reply cloud/README.md:15]: These will be re-provided in a later pull request

File: cloud/README.md [hunk 2]
```diff
@@ -2,15 +2,84 @@
...
+Output:
+
+```text
+api.test.cloud.xxx.streamnative.dev
+```
+
+A `SERVICE_URL` is a combination of protocol, hostname and port, so an example of a complete `SERVICE_URL` is as follows:
+
+
+```text
```
[the remark this answers is not in the document]
Comment (wolfstudy) [reply cloud/README.md:49]: Here is just an output example, users can get the real hostname through the above command

File: cloud/pulsarctl/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Overview
```
[the remark this answers is not in the document]
Comment (wolfstudy) [reply cloud/pulsarctl/README.md:1]: Cool, will add it in next pull request

File: cloud/README.md [hunk 4]
```diff
@@ -2,15 +2,87 @@
...
+For the OAuth2, it needs to obtain the specified options from the Pulsar cluster and OAuth2 services. [How to get OAuth2 options](#How to get OAuth2 options) explains how you get these options.
 
 Currently, we support the following three languages to connect through OAuth2:
 
 - Java
 - Go
 - CPP
 
-For clients in other languages, you can connect through token, reference to [here](https://pulsar.apache.org/docs/en/security-tls-transport/#client-configuration).
+The following clients and Pulsar CLI tools are supported to connect to cluster through the Token. And [How to get Token options](#How to get Token options) explains how you get these options.
```
Comment (Huanli-Meng) [inline cloud/README.md:15]: what about the websocket client

File: cloud/README.md [hunk 5]
```diff
@@ -2,15 +2,87 @@
...
+- CPP
+- Python
+- CSharp(TODO)
+- NodeJS
+- pulsarctl
+- pulsar-admin
+- pulsar-client
+- pulsar-perf
+
+## How to get Token options
```
Comment (Huanli-Meng) [inline cloud/README.md:28]: i just curious why the "Token" uses a uppercase letter "T"

File: cloud/README.md [hunk 6]
```diff
@@ -2,15 +2,87 @@
...
+$ snctl get pulsarclusters [CLUSTER_NAME] -n [NAMESPACE] -o json | jq '.spec.serviceEndpoints[0].dnsName'
+```
+
+Output:
+
+```text
+api.test.cloud.xxx.streamnative.dev
+```
+
+A `WEB_SERVICE_URL` is a combination of protocol, hostname and port, so an example of a complete `WEB_SERVICE_URL` is as follows:
```
Comment (Huanli-Meng) [inline cloud/README.md:70]: port ID?

File: cloud/README.md [hunk 7]
```diff
@@ -2,15 +2,87 @@
...
+$ snctl get pulsarclusters [CLUSTER_NAME] -n [NAMESPACE] -o json | jq '.spec.serviceEndpoints[0].dnsName'
+```
+
+Output:
+
+```text
+api.test.cloud.xxx.streamnative.dev
+```
+
+A `SERVICE_URL` is a combination of protocol, hostname and port, so an example of a complete `SERVICE_URL` is as follows:
```
Comment (Huanli-Meng) [inline cloud/README.md:48]: port ID?

File: cloud/csharp/README.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
+# Overview
```
Comment (Huanli-Meng) [inline cloud/csharp/README.md:1]: mode contents required?

File: cloud/node/README.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+# Prerequisites
+
+Pulsar Node.js client library is based on the C++ client library. Follow the instructions for
+[C++ library](https://pulsar.apache.org/docs/en/client-libraries-cpp/) for installing the binaries through
+[RPM](https://pulsar.apache.org/docs/en/client-libraries-cpp/#rpm),
+[Deb](https://pulsar.apache.org/docs/en/client-libraries-cpp/#deb) or
+[Homebrew packages](https://pulsar.apache.org/docs/en/client-libraries-cpp/#macos).
+
+> #### Note
+> you will need to install not only the pulsar-client library but also the pulsar-client-dev library.
```
Comment (Huanli-Meng) [inline cloud/node/README.md:14]:
```suggestion
> You need to install both the pulsar-client library and the pulsar-client-dev library.
```

File: cloud/node/README.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+Also, this library works only in Node.js 10.x or later because it uses the
+[node-addon-api](https://github.com/nodejs/node-addon-api) module to wrap the C++ library.
+
+## Install 
+
+### Please install pulsar-client in your project:
```
Comment (Huanli-Meng) [inline cloud/node/README.md:21]:
```suggestion
### Install pulsar-client in your project
```

File: cloud/pulsar-admin/README.md [hunk 11]
```diff
@@ -0,0 +1,22 @@
+# Overview
+
+The `pulsar-admin` is a CLI tool written by Java language for the Apache Pulsar project.
```
Comment (Huanli-Meng) [inline cloud/pulsar-admin/README.md:3]:
```suggestion
The `pulsar-admin` is a CLI tool written in Java language for the Apache Pulsar project.
```

File: cloud/pulsar-admin/README.md [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+
+# Prerequisites
+
+- Pulsar broker 2.7.0-742fc5c9b+
+
+> You can get this tarball from [bintray](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When the 2.6.1 release is complete, you can also use the official 2.6.1 version.
```
Comment (Huanli-Meng) [inline cloud/pulsar-admin/README.md:9]:
```suggestion
> You can get this tarball from [binary](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When Pulsar 2.6.1 is released, you can also use the official 2.6.1 version.
```

File: cloud/pulsar-admin/README.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+
+# Usage
+
+The `pulsar-admin` supports to connect to Pulsar cluster through Token, the example as follows:
```
Comment (Huanli-Meng) [inline cloud/pulsar-admin/README.md:13]:
```suggestion
The `pulsar-admin` supports to connect to Pulsar cluster through Token, as shown below:
```

File: cloud/pulsar-client/README.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
+# Overview
+
+The `pulsar-client` and `pulsar-perf` are CLI tool written by Java language for the Apache Pulsar project.
```
Comment (Huanli-Meng) [inline cloud/pulsar-client/README.md:3]:
```suggestion
The `pulsar-client` and `pulsar-perf` are CLI tool written in Java language for the Apache Pulsar project.
```

File: cloud/pulsar-client/README.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+
+# Prerequisites
+
+- Pulsar broker 2.7.0-742fc5c9b+
+
+> You can get this tarball from [bintray](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When the 2.6.1 release is complete, you can also use the official 2.6.1 version.
```
Comment (Huanli-Meng) [inline cloud/pulsar-client/README.md:9]:
```suggestion
> You can get this tarball from [bunary](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When Pulsar 2.6.1 is released, you can also use the official 2.6.1 version.
```

File: cloud/pulsar-client/README.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+
+# Usage
+
+The `pulsar-client` supports to connect to Pulsar cluster through Token, the example as follows:
```
Comment (Huanli-Meng) [inline cloud/pulsar-client/README.md:13]:
```suggestion
The `pulsar-client` supports to connect to Pulsar cluster through Token, as shown below:
```

File: cloud/pulsarctl/README.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+
+# Prerequisites
+
+- pulsarctl 0.5.0+
+
+More information reference to [here](https://github.com/streamnative/pulsarctl/blob/master/README.md).
+
+# Usage
+
+The `pulsarctl` supports to connect to Pulsar cluster through Token, the example as follows:
```
Comment (Huanli-Meng) [inline cloud/pulsarctl/README.md:13]:
```suggestion
The `pulsarctl` supports to connect to Pulsar cluster through Token, as shown below:
```

File: cloud/README.md (hunk 4 above)
Comment (wolfstudy) [reply cloud/README.md:15]: next pull request, I need to investigate

File: cloud/README.md (hunk 5 above)
Comment (wolfstudy) [reply cloud/README.md:28]: We can change it to lower case

File: cloud/README.md (hunk 6 above)
Comment (wolfstudy) [reply cloud/README.md:70]: What do you mean?

File: cloud/pulsar-admin/README.md (hunk 12 above)
Comment (wolfstudy) [reply cloud/pulsar-admin/README.md:9]: `bintray` is the name of a website, not typo.

File: cloud/pulsarctl/README.md [hunk 18]
```diff
@@ -0,0 +1,59 @@
...
+
+# Prerequisites
+
+- pulsarctl 0.5.0+
+
+More information reference to [here](https://github.com/streamnative/pulsarctl/blob/master/README.md).
+
+# Usage
+
+## Connect By Token
```
Comment (Huanli-Meng) [inline cloud/pulsarctl/README.md:13]:
```suggestion
## Connect to cluster through Token
```

File: cloud/pulsar-client/README.md [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+The `pulsar-perf` supports to connect to Pulsar cluster through Token, as shown below:
+
+```shell script
+./bin/pulsar-perf \
+    --service-url SERVICE_URL \
+    --auth-params AUTH_PARAMS \
+    produce -r 1000 -s 1024 test-topic
+```
+
+How to get the `SERVICE_URL` and `AUTH_PARAMS` fields, please reference to **how to get Token options**.
```
Comment (Huanli-Meng) [inline cloud/pulsar-client/README.md:31]:
```suggestion
For details about how to get the `SERVICE_URL` and `AUTH_PARAMS` fields, please reference to **how to get Token options**.
```

File: cloud/pulsarctl/README.md [hunk 20]
```diff
@@ -0,0 +1,59 @@
...
+  --client-id 0Xx...hyYyxeny \
+  --audience audience-path \
+  --key-file /path/to/private/key
+```
+
+### Use Go Admin API
+
+The `pulsarctl` itself provides the function of the Go Admin API. You can use the interface of the Admin API to use the function of OAuth2 to establish a connection with the Pulsar cluster. 
+
+How to get the `keyFile`, `IssuerEndpoint`, `Audience` and `ClientID` fields, please reference to **how to get OAuth2 options**.
```
Comment (Huanli-Meng) [inline cloud/pulsarctl/README.md:59]:
```suggestion
For details about how to get the `keyFile`, `IssuerEndpoint`, `Audience` and `ClientID` fields, refer to **how to get OAuth2 options**.
```

File: cloud/pulsar-admin/README.md [hunk 21]
```diff
@@ -0,0 +1,22 @@
...
+The `pulsar-admin` supports to connect to Pulsar cluster through Token, as shown below:
+
+```shell script
+./bin/pulsar-admin \
+    --url WEB_SERVICE_URL \
+    --auth-params AUTH_PARAMS \
+    tenants list
+```
+
+How to get the `WEB_SERVICE_URL` and `AUTH_PARAMS` fields, please reference to **how to get Token options**.
```
Comment (Huanli-Meng) [inline cloud/pulsar-admin/README.md:22]:
```suggestion
For details about how to get the `WEB_SERVICE_URL` and `AUTH_PARAMS` fields, refer to **how to get Token options**.
```

and use the cross reference about [how to get Token options](). same as below.

File: cloud/pulsarctl/README.md [hunk 22]
```diff
@@ -0,0 +1,59 @@
...
+The `pulsarctl` supports to connect to Pulsar cluster through Token, the example as follows:
+
+```shell script
+pulsarctl \
+    --admin-service-url WEB_SERVICE_URL \
+    --token AUTH_PARAMS \
+    tenants list
+```
+
+How to get the `WEB_SERVICE_URL` and `AUTH_PARAMS` fields, please reference to **how to get Token options**.
```
Comment (Huanli-Meng) [inline cloud/pulsarctl/README.md:24]:
```suggestion
For details about how to get the `WEB_SERVICE_URL` and `AUTH_PARAMS` fields, refer to **how to get Token options**.
```

File: cloud/pulsarctl/README.md [hunk 23]
```diff
@@ -0,0 +1,59 @@
...
+
+## Connect By OAuth2
```
Comment (Huanli-Meng) [inline cloud/pulsarctl/README.md:26]:
```suggestion
## Connect to cluster through OAuth2
```

File: cloud/pulsarctl/README.md [hunk 24]
```diff
@@ -0,0 +1,59 @@
...
+
+The `pulsarctl` supports to connect to Pulsar cluster through OAuth2, provide the following two ways:
+
+### Use CLI Tool
+
+The `pulsarctl` supports to connect to Pulsar cluster through OAuth2, the example as follows:
```
Comment (Huanli-Meng) [inline cloud/pulsarctl/README.md:32]:
```suggestion
The `pulsarctl` supports to connect to Pulsar cluster through OAuth2, as shown below:
```

File: cloud/pulsarctl/README.md [hunk 25]
```diff
@@ -0,0 +1,59 @@
...
+
+1. Activate an account with keyfile.
+
+```shell script
+$ pulsarctl oauth2 activate \
+  --issuer-endpoint https://oauth.service \
+  --client-id 0Xx...hyYyxeny \
+  --audience audience-path \
+  --key-file /path/to/private/key
+```
```
Comment (Huanli-Meng) [inline cloud/pulsarctl/README.md:42]: adjust the indent of the code example. same as below.

**Pull request merged**